### PR TITLE
8 display sidebar trails

### PR DESCRIPTION
**Pull request by gwan93**
Functionality to list the trail names in the sidebar

### CODE REVIEW

File: src/components/Sidebar.js [hunk 1]
```diff
@@ -1,33 +1,52 @@
...
+  const [trailNames, setTrailNames] = useState([]);
+
   useEffect(() => {
-    getTrails()
-    .then(data => {
-      data.trails.forEach(trail => {
-        console.log(trail.name)
-      })
-    })
+    getTrails().then((data) => {
```
Comment (parmvirthind) [inline src/components/Sidebar.js]: can make use of object destructuring here:
```suggestion
    getTrails().then(({ trails }) => {
```

File: src/components/Sidebar.js [hunk 2]
```diff
@@ -1,33 +1,52 @@
...
+      const namesArr = [];
+      data.trails.forEach((trail) => namesArr.push(trail.name));
+      setTrailNames(namesArr);
```
Comment (parmvirthind) [inline src/components/Sidebar.js]: I don't think creating an array specific to names is necessary. I think it is okay to just do:
```suggestion
     setTrails(trails)
```
That way in the future if we wanted to display other info in addition to the name, we will already have access to those properties

**Review by parmvirthind — APPROVED** [review]
minor comments

**Comment by gwan93** [comment]
Agree with both comments. Will implement

**Pull request merged**